playbooks/files/rax-maas/plugins/process_check_container.py

```diff
@@ -130,9 +130,9 @@ def check_process_running(process_names, container_name=None):
             # a callable method.
             cmdline_check = getattr(proc, "cmdline", None)
             if callable(cmdline_check):
-                cmdlines.append(map(os.path.basename, proc.cmdline()))
```

Comment:
I don't think the change here is needed. 
the map of os.path.basename will just transform a list,  something like ["/usr/bin/hummingbird"] to ["hummingbird"].